molecule/api.py

```diff
@@ -1,6 +1,11 @@
+try:
+    from functools import lru_cache
+except ImportError:
+    from backports.functools_lru_cache import lru_cache
 import pkg_resources
 
 
+@lru_cache()
 def molecule_drivers(as_dict=False):
```

Comment:
Just passing by but wasn't there something like the "memoizer" already built in for caching lookups?

Otherwise, it's a bit meh to have to add a dependency just for this ...

Reply:
@decentral1se What can I say,.... I was not aware about existence of `util.memoize`. I will use it as we already have it.

Usually I am against adding depedencies but lru_cache is part of python since 3.3 and the only additional requirement would be the backport for 2.7 --- which I find better than having in-house reimplementation of the same feature. Anyway, I will update it to use memoize.